tests/ui.rs

```diff
@@ -153,14 +183,13 @@ fn run_tests(
     }
 
     // Handle command-line arguments.
-    let args = ui_test::Args::test()?;
-    let default_bless = env::var_os("RUSTC_BLESS").is_some_and(|v| v != "0");
-    config.with_args(&args, default_bless);
-    if let OutputConflictHandling::Error(msg) = &mut config.output_conflict_handling {
-        *msg = "./miri test --bless".into();
-    }
+    let mut args = ui_test::Args::test()?;
+    args.bless = env::var_os("RUSTC_BLESS").is_some_and(|v| v != "0");
```

Comment:
Should we use `|=` so that if the ui_test parser sees a `--bless` we don't just ignore that?